common/lib/xmodule/xmodule/util/duedate.py

```diff
@@ -21,3 +21,15 @@ def get_extended_due_date(node):
     if not extended or extended < due_date:
         return due_date
     return extended
+
+
+def escape_export_name(name, replace_with='_'):
```

Comment:
We should rename this method as it's not dealing with export only. What about `escape_invalid_characters` ?